#!/usr/bin/env python
PACKAGE = "hough_circle"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("canny_thres", int_t, 0, "Canny threshold", 200,1,300)
gen.add("accumulator_thres", int_t, 0, "Accumulator threshold",120,1,200)

exit(gen.generate(PACKAGE, "canny_edge_my_face", "Threshold"))
